receiver/otlpreceiver/config.md

```diff
@@ -0,0 +1,91 @@
+# "otlp" Receiver Reference
+
+Config defines configuration for OTLP receiver.
+
+
+### Config
```

Comment:
`otlpreceiver.Config`? If all the others have a package like title?

Reply:
Was originally there, but I removed the package in the title by request 🙂 https://github.com/open-telemetry/opentelemetry-collector/pull/3100#discussion_r626785756 Happy to change it back -- either way is fine with me.